src/h5web/visualizations/heatmap/ColorBar.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { range } from 'lodash-es';
+import { ScaleSequential, scaleLinear } from 'd3-scale';
+import { AxisRight } from '@vx/axis';
+import useMeasure from 'react-use-measure';
+import { adaptedNumTicks } from './utils';
+import styles from './HeatmapVis.module.css';
+
+interface Props {
+  className: string;
+  scale: ScaleSequential<string>;
+}
+
+function ColorBar(props: Props): JSX.Element {
+  const { className, scale } = props;
+  const interpolator = scale.interpolator();
+  const gradientColors = range(0, 1.1, 0.1)
+    .map(i => `${interpolator(i)}`)
+    .reduce((acc, val) => `${acc},${val}`);
+
+  const [gradientRef, { height: gradientHeight }] = useMeasure();
+
+  return (
+    <div className={className}>
+      <div className={styles.colorBar}>
+        <div
+          ref={gradientRef}
+          className={styles.gradient}
+          style={{
+            background: `linear-gradient(${gradientColors})`,
+          }}
+        />
+        <svg className={styles.colorBarAxis}>
```

Comment:
May as well put `height={gradientHeight}` here, and a static `width`. It's better to have explicit dimensions on an SVG element. You can also remove `width: min-content` in the CSS, since this actually results in the SVG having a width of `0`.